gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -262,7 +270,7 @@ local function CreateEquipmentList(t)
 	if IsValid(dlist) then
 		dlist:Clear()
 	else
-		TraitorMenuPopup() --TODO Check
+		TraitorMenuPopup() -- TODO Check
```

Comment:
Todo?

Reply:
not by me ;)

Reply:
yes, but didn't we decied on moving this kind of stuff to trello?

Reply:
Feel free doing so :D